tests/test_web_server.py

```diff
@@ -102,10 +104,54 @@
     cli = await aiohttp_client(server)
     resp = await cli.get('/path/to')
     assert resp.status == 500
+    assert resp.headers['Content-Type'].startswith('text/plain')
 
     txt = await resp.text()
-    assert "<h2>Traceback:</h2>" in txt
+    assert "Traceback:" in txt
 
     logger.exception.assert_called_with(
         "Error handling request",
         exc_info=exc)
+
+
+async def test_raw_server_html_exception(aiohttp_raw_server, aiohttp_client):
+    exc = RuntimeError("custom runtime error")
+
+    async def handler(request):
+        raise exc
+
+    logger = mock.Mock()
+    server = await aiohttp_raw_server(handler, logger=logger, debug=False)
```

Comment:
Currently, tests are failing with `ServerDisconnectedError` for this test only. I'm not sure why?

`ServerDisconnectedError` doesn't happen with `debug=True` or if I remove the `Accept` header.

Reply:
Sorry, I don't remember details

Reply:
Fixed. 

my mistake :nerd_face: :gun: